jmclient/jmclient/wallet.py

```diff
@@ -1013,6 +979,381 @@ def close(self):
     def __del__(self):
         self.close()
 
+class DummyKeyStore(btc.KeyStore):
+    @classmethod
+    def from_iterable(cls, iterable, **kwargs):
```

Comment:
This method seems to be identical to the one in parent class

Reply:
Yes, same as previous case, a result of setting up the initial experiment, removed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536/commits/9e9ae9e287e2d6e97ea9d53ca628d79fd8e81453